server/db/driver/pg/matches.go

```diff
@@ -50,7 +51,7 @@ func userMatches(ctx context.Context, dbe *sql.DB, tableName string, aid account
 		var baseRate, quoteRate sql.NullInt64
 		var takerSell sql.NullBool
 		var takerAddr, makerAddr sql.NullString
-		err := rows.Scan(&m.ID, &takerSell, &m.Active,
+		err := rows.Scan(&m.ID, &m.Active, &takerSell,
```

Comment:
These booleans were switched.  Caught by the `AllActiveUserMatches` test that now uses this function.